Libraries/LibWeb/TrustedTypes/TrustedTypePolicy.cpp

```diff
@@ -10,6 +10,7 @@
 #include <LibJS/Runtime/Realm.h>
 #include <LibWeb/Bindings/Intrinsics.h>
 #include <LibWeb/TrustedTypes/TrustedHTML.h>
+#include <LibWeb/TrustedTypes/TrustedScript.h>
```

Comment:
There are currently 2 "LibWeb: Implement TrustedScript type"  commits. It looks like these should be merged into a single commit?

Reply:
With all the rebasing and edited I made I mistake

I managed to fix it let me know if it is okay now :)